Index: pkg-tigervnc/unix/xserver/hw/vnc/Makefile.am
===================================================================
--- pkg-tigervnc.orig/unix/xserver/hw/vnc/Makefile.am
+++ pkg-tigervnc/unix/xserver/hw/vnc/Makefile.am
@@ -1,5 +1,5 @@
 TIGERVNC_SRCDIR=${top_srcdir}/../..
-TIGERVNC_BUILDDIR=${TIGERVNC_SRCDIR}
+TIGERVNC_BUILDDIR=${top_builddir}/../..
 
 RFB_LIB=$(TIGERVNC_BUILDDIR)/common/rfb/librfb.la
 RDR_LIB=$(TIGERVNC_BUILDDIR)/common/rdr/librdr.la
